src/preprocess/LIDC-IDRI_dcm2nifty.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import argparse
+import nibabel as nib
+from pathlib import Path
+
+import dicom2nifti
+
+
+def main(args):
+    data_dir = args.data_dir
+    output_dir = args.output_dir
+    
+    if output_dir.exists() is False:
+        output_dir.mkdir(parents=True)
+    
+    patient_paths = sorted(list((data_dir).iterdir()))
```

Comment:
It doesn't cause any errors here since there are only folders in the dataset folder.
But I'll modify the code for the clear representation and avoiding any potential errors.